profiler/shakesapp/shakesapp_main.go

```diff
@@ -56,18 +57,18 @@ func main() {
 	shakesapp.RegisterShakespeareServiceServer(server, shakesapp.NewServer())
 	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", *port))
 	if err != nil {
-		log.Fatalf("failed to listen: %v", err)
+		glog.Exitf("failed to listen: %v", err)
```

Comment:
nit: failed -> Failed

Please make a pass to check that log messages start with capital letter, error strings - with lower.

Reply:
Done.